spec/latest/core/index.html

```diff
@@ -1039,6 +1039,44 @@ <h3>Templated Links</h3>
       }
       -->
     </pre>
+
+    <p>Iri expansion should be performed with respect to the specification
+      behind the iri template type (which is RFC 6570 by default), and the
+      product of this process SHOULD be an Iri. In may happen that this
+      product is a relative Uri. In such a case client by default SHOULD
+      stick to RFC 3986. It is due to fact that most RDF serialization
+      allowing other that absolute Iris will behave that way and the result
+      of Iri template expansion should be compliant. Still, it may be
+      preferred to use other base Uri for the expansion process, which
+      makes a <i>resolveRelativeTo</i> term useful. It allows to switch the
+      Iri template expansion algorithm so the base Uri is established using
+      current link context, which is a subject of the relation pointing to
+      <i>IriTemplate</i> instance. In case that subject is a relative Uri,
+      default behavior SHOULD be used as fallback.</p>
+
+    <p>The example below allows to hook the product of an Iri template
+      expansion to the <i>http://api.example.com/an-issue/</i> resource
+      as it's base Uri, which further enables the <i>some:operation</i> to
+      be moved to i.e. Api documentation level rather to inline it.</p>
```

Comment:
I made a couple of changes to the wording and uppercased abbreviations here as well.

```suggestion
    <p>The example below allows to make the product of an IRI template
      expansion relative to the <i>http://api.example.com/an-issue/</i> resource
      by using it as its base URI, which further enables the
      <i>some:operation</i> to be moved to i.e. API documentation level rather
      to inline it.</p>
```